lib/src/collection/impl/set.dart

```diff
@@ -51,10 +51,9 @@ class DartSet<T> extends Object implements KtSet<T> {
     if (other.hashCode != hashCode) return false;
     if (other is KtSet<T>) {
       return containsAll(other);
-    } else if (other is KtSet) {
+    } else {
       return other.containsAll(this);
     }
-    return false;
```

Comment:
Needs to be double checked, I'm not sure if this is correct

Reply:
From [pub.dev](https://pub.dev/packages/kt_dart/score):

<img width="792" alt="Screen Shot 2022-09-06 at 18 52 25" src="https://user-images.githubusercontent.com/43643339/188694586-500ed9e4-1a6a-45b5-a3b2-e0a26f191de4.png">